src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -17,6 +17,7 @@
     }
   },
   "networkAddresses": {
+    "8008": ["canonical", "eip155"],
```

Comment:
Can you make it so the chain IDs are ordered?

I fixed the test so that it properly reports the error in #1194